### ISSUE TITLE
JF Web 10.6.3 on Tizen 5.5 playback sub-menu issue

### ISSUE THREAD

**Issue by Nazar78** [jellyfin/jellyfin-tizen#50]
With JF Web 10.6.3 or the master branch, I can no longer close the full screen sub-menu during playback session on both emu and Tizen 5.5.

Steps taken to reproduce on both emu 5.5 and RU8000:

1. Connect to JF server via its IP:8096.

2. Play any video.

3. Click on the bottom panel subtitle or settings icon.

4. Click on any of the opened full screen sub-menus, there's a slight loading circle animation then nothing happens. Even physical back button couldn't close it.

Reverted back to JF Web 10.6.2 and it works normally.

**Comment by dmitrylyzo** [comment]
Dialogs on `master` are broken: https://github.com/jellyfin/jellyfin-web/pull/1645#discussion_r471739855
10.6.3 works for me. Maybe caching issue in your case?

**Comment by Nazar78** [comment]
I connect directly on home TV without proxy since I still have that proxy issue. Fortunately the reverse proxy is only being used by several other Android TVs remotely.

I actually first installed the 10.6.3 when I noticed the issue, usually I'll go for the release instead of master. Only just now I tried again building both master/10.6.3 and tested them on both emu and TV, the sub-menu won't close. Finally rebuilt and installed the 10.6.2 only then it works.
